# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1458601011 25200
#      Mon Mar 21 15:56:51 2016 -0700
# Node ID 6bf7f41afea68eebe52910d3dc9b1ff6e36a679d
# Parent  79aa496c6d47d8ad14a69f96ae695146ee32b397
* configure.ac: Correct typo of "gnuplot" in warning message

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -3556,17 +3556,17 @@ if test $ENABLE_64 = yes; then
   AC_MSG_WARN([libraries to use 8 byte signed integers for array indexing.])
   warn_msg_printed=true
 fi
 
 if test $opengl_graphics = no; then
   AC_MSG_WARN([])
   AC_MSG_WARN([I didn't find the libraries needed to compile Octave])
   AC_MSG_WARN([with support for OpenGL graphics.  Creating plots is])
-  AC_MSG_WARN([still possible if you have gnpulot installed.])
+  AC_MSG_WARN([still possible if you have gnuplot installed.])
   warn_msg_printed=true
 fi
 
 if test $ENABLE_ATOMIC_REFCOUNT = no; then
   AC_MSG_WARN([])
   AC_MSG_WARN([atomic reference counting disabled.])
   AC_MSG_WARN([This feature allows access to Octave data safely from])
   AC_MSG_WARN([another thread, and is required when using the GUI's Qt toolkit])
